src/components/cms/SkillFeedbackPage/SkillFeedbackPage.js

```diff
@@ -540,18 +615,37 @@ class SkillFeedbackPage extends Component {
     let feedbackCardsElement = null;
     feedbackCardsElement = (
       <div>
-        {accessToken && !userFeedback ? (
+        {accessToken && !userRating ? (
           <div>
             <SubTitle size="1rem">
-              {`Write your invaluable feedback with
+              {`Write your invaluable feedback and rating with
             ${this.skillName} on SUSI.AI`}
             </SubTitle>
             <div>
               <Timestamp>
                 <FormControl fullWidth={true}>
+                  <div
+                    style={{
+                      textAlign: 'center',
+                      margin: '1.5rem',
+                    }}
+                  >
+                    <Rating
+                      rating={this.state.rating}
+                      widgetRatedColors="#ffbb28"
```

Comment:
These props are present in almost all scenarios. Please put them under the common component.